### PR TITLE
Add http signed relay gateway example to simplify testing

### PR DESCRIPTION
**Pull request by Siegrift**
https://api3dao.atlassian.net/browse/AN-557

### CODE REVIEW

**Review by Siegrift — COMMENTED** [review]
I just tested the new updates (rename of the handler) and got this response:

```
{"data":"{\"timestamp\":\"1645786263\",\"value\":\"0x00000000000000000000000000000000000000000000000000000008fd651ac0\"}","signature":"0xbc017f87fe39795e5ab89511b4a76185ee2426316a259b4dc1448fea4dc7991373ec35b428529f25a25067a05d7dd6b2abd37b8441e9051a17631fa0d78bde751c"}
```

The `data` contains stringified response (which is not nice), but that will be resolved in https://github.com/api3dao/airnode/pull/871

File: packages/airnode-examples/integrations/coingecko-signed-data/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Coingecko-signed-data example integration
+
+This example contains the same API integration as `../coingecko`.
+
+The only difference is that this integration allows the endpoint to return signed data that can be submitted on chain by
+the to update a beacon.
```
Comment (vponline) [inline packages/airnode-examples/integrations/coingecko-signed-data/README.md:6]: This sentence seems to be missing a word :thinking:

Comment (amarthadan) [reply packages/airnode-examples/integrations/coingecko-signed-data/README.md:6]: Yes, looks like it. Shorten it to just
`The only difference is that this integration allows the endpoint to return signed data that can be then submitted on chain.`

**Review by amarthadan — APPROVED** [review]
I am a bit confused by the number and content of your commits. It should have been just one commit with a few lines changed in the `airnode-examples` package. What happened there? :smile:

But the result seems fine and it's building so the renaming probably went well. I'll leave the testing up to @dcroote.

**Comment by vponline** [comment]
Ok this was a misunderstanding on my part, I renamed `signedData` to `httpSignedData` (and affected variables/functions) everywhere. My bad :pray:

File: packages/airnode-examples/contracts/coingecko-signed-data/Requester.sol [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    function fulfill(bytes32 requestId, bytes calldata data)
+        external
+        onlyAirnodeRrp
+    {
+        require(incomingFulfillments[requestId], "No such request made");
+        delete incomingFulfillments[requestId];
+        int256 decodedData = abi.decode(data, (int256));
+        fulfilledData[requestId] = decodedData;
+    }
+}
```
Comment (vponline) [inline packages/airnode-examples/contracts/coingecko-signed-data/Requester.sol:41]: I added this (copy of Coingecko)

**Review by amarthadan — APPROVED** [review]

**Pull request merged**